src/model_module/environment.py

```diff
@@ -89,7 +93,7 @@ def __init__(
 
         self.device = device
         self.render_mode = render_mode
-        self.max_layers = max_layers
+        self.max_layers = MAX_LAYERS
```

Comment:
Currently the max_layers argument in the constructor is ignored, which is not ideal. So either it should be removed or the follow change should be made:

```suggestion
        self.max_layers = max_layers
```